schemas/groups/propulsion.json

```diff
@@ -82,6 +65,109 @@
       "units": "Pa"
     },
 
+    "alternatorVoltage": {
+      "description": "Alternator voltage",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "V"
+    },
+
+    "engineTime": {
+      "description": "Total running time for engine",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "s"
+    },
+
+    "engineTemperature": {
+      "description": "Engine temperature",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "K"
+    },
+
+    "coolantPressure": {
+      "description": "Coolant pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "boostPressure": {
+      "description": "Engine boost (turbo, supercharger) pressure",
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "units": "Pa"
+    },
+
+    "transmission": {
+      "type": "object",
+      "properties": {
+        "gear": {
+          "type": "object",
+          "properties": {
+            "value": {
+              "enum": ["Forward", "Neutral", "Reverse", "Fault"]
+            }
+          }
+        },
+        "oilTemperature": {
+          "description": "Oil temperature",
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "units": "K"
+        },
+        "oilPressure": {
+          "description": "Oil pressure",
+          "$ref": "../definitions.json#/definitions/numberValue",
+          "units": "Pa"
+        }
+      }
+    },
+
+    "drive": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "enum": ["saildrive", "shaft", "outboard"]
+        },
+        "trimState": {
```

Comment:
My thought was a rudder is a passive system where this is actively thrusting in a specific direction, especially for a maneuvering unit like a bow thruster. One difference is that the thrust from a rudder is a function of boat speed, whereas thrust from the drive is a function of shaft speed.

For example, a small dinghy with an outboard that completely rotates has no rudder and steers purely based on drive thrust angle; it has no steering without drive thrust, same as a personal watercraft/jetski. Same principle applies to a large cruise ship or ferry with azimuth drives.